salesagility/SuiteCRM
# Fixed #3594 - AOR Report- Datetime is not converted to UTC timezone thus providing inaccurate report

## Description

When you create a report with date parameter of type 'Period' with any value, then the resultant report is not accurate.

If you log the query, which gets generated for this report, then you can see that the datetime doesn't get converted to UTC timezone and because of this, inaccurate reports gets generated.

So, to fix this issue, changed timezone of date parameter to UTC/GMT.

Steps to reproduce:

1. Create a new **Report** with any datetime parameter as a condition and type as a _Period_ and value as a _Last Quarter_
2. Log the query and check the datetime parameter should be converted to UTC/GMT timezone.


Issue no. #3594 

## Motivation and Context

It converts datetime to UTC/GMT timezone.

## How To Test This

Steps to reproduce:

1. Create a new **Report** with any datetime parameter as a condition and type as a _Period_ and value as a _Last Quarter_
2. Log the query and check the datetime parameter should be converted to UTC/GMT timezone.

## Types of changes


### Final checklist
